pkg/datapath/linux/routing/migrate_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2020 Authors of Cilium
+// Copyright 2020-2021 Authors of Cilium
```

Comment:
Totally nitpicky - shouldn't this be `2016-2021`?

Reply:
The original 2016 was copy-pasta from another file. `migrate_test.go` is brand new is 2020-2021 :)